config/environment.js

```diff
@@ -21,6 +23,21 @@ module.exports = function (environment) {
       // Here you can pass flags/options to your application instance
       // when it is created
     },
+
+    'mapbox-gl': {
+      accessToken: 'pk.eyJ1IjoibGVlbmEtd2ViIiwiYSI6ImNrb2k4dTNzajB1MWsyb25zcDQ5MDh5ZTMifQ.KXpvP9jclC3Ry6oa4vy7OA',
```

Comment:
This is hard-coding. Same as you've used ENV how do you remove this hard coding and manage the `accessToken` and possible other env variables for different environments like staging and production?

Reply:
We can use `ember-cli-dotenv`, using addon we can manage env variables for diffrent environments

Reply:
:100: